### ISSUE TITLE
Emphasize Ember.Service life span

### ISSUE THREAD

**Issue by Tonkpils** [emberjs/guides#1436]
I saw a question today about whether Services were persisted across full page refreshes. Reading the guides for `Ember.Service`, it states that the "service will persist until the application exists" but it's further ahead in reading and separated from a statement that says the service is a long lived object.

It would help to move the sentence about the service being persisted until the application exits closer to the one talking about the lifespan of the object

I'd be glad to make a PR if this is a change you'd like to accept. Just following the contributing guides 😃

**Comment by locks** [comment]
Hey @Tonkpils ;)

So, I'm a tad ambivalent on this. On the one hand, I do understand your initial confusing, and I want to make sure the documentation is as clear as possible. On the flip side, I include knowing that the JavaScript gets wiped when doing a page refresh in the "working knowledge" of browser stuff. I wonder if part of the problem is thinking that `Ember.Service` might be "special" and not a regular JavaScript object?

tl;dr please make the PR, I'm thinking aloud above to get my thoughts down in words for later reference.

**Comment by Tonkpils** [comment]
To be honest it may just be the wording on the `long-lived object` may have swayed my thinking to think that `Ember.Service` is a special object. When I saw the question, I originally thought that the object would be wiped on the refresh, but the documentation made me doubt my original thought...that's why I think maybe being specific would help others clear up similar concerns

Here's I ran into http://stackoverflow.com/questions/37356050/ember-service-is-lost-on-reload/37356252

I will make the PR and we can chat to see if it does make sense  😄
